templates/common/_base/files/kubelet-auto-sizing.yaml

```diff
@@ -46,15 +46,16 @@ contents:
         total_cpu=$(getconf _NPROCESSORS_ONLN)
         recommended_systemreserved_cpu=0
         if (($total_cpu <= 1)); then # 6% of the first core
-            recommended_systemreserved_cpu=$(echo $total_cpu 0.60 | awk '{print $1 * $2}')
+            recommended_systemreserved_cpu=$(echo $total_cpu 0.06 | awk '{print $1 * $2}')
             total_cpu=0
         else
             recommended_systemreserved_cpu=0.06
             total_cpu=$((total_cpu-1))
         fi
         if (($total_cpu <= 1)); then # 1% of the next core (up to 2 cores)
-            recommended_systemreserved_cpu=$(echo $recommended_systemreserved_cpu $(echo $total_cpu 0.10 | awk '{print $1 * $2}') | awk '{print $1 + $2}')
-            total_cpu=0 else
+            recommended_systemreserved_cpu=$(echo $recommended_systemreserved_cpu $(echo $total_cpu 0.01 | awk '{print $1 * $2}') | awk '{print $1 + $2}')
+            total_cpu=0
+        else
```

Comment:
`else` in this line was misplaced.

Reply:
Also, there was a typo on line 56 where we wanted to get 1% of CPU but we were getting 10%.